services/movie.py

```diff
@@ -0,0 +1,42 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+    if genres_ids and actors_ids:
+        return queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    if genres_ids:
+        return queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        return queryset.filter(actors__id__in=actors_ids)
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)

```

Comment:
The `get_movie_by_id` function uses `get()`, which will raise a `DoesNotExist` exception if the movie is not found. Consider using `get_object_or_404()` if this function is used in a view, or handle the exception to improve robustness.